src/common/style/vanilla/forms.scss

```diff
@@ -0,0 +1,107 @@
+@import 'vanilla-framework/scss/settings';
+@import 'vanilla-framework/scss/base_forms.scss';
+
+@mixin theme-p-forms {
```

Comment:
Where do these theme styles come? Is it dashboard theme?

It would be good to have a comment about it here to know where to look for it in case of updates.